sys/net/gnrc/Makefile.dep

```diff
@@ -135,6 +135,7 @@ ifneq (,$(filter gnrc_netif,$(USEMODULE)))
   USEMODULE += netif
   USEMODULE += l2util
   USEMODULE += fmt
+  USEMODULE += ztimer_msec
```

Comment:
```suggestion
  USEMODULE += ztimer_msec
  USEMODULE += ztimer_usec
```

Reply:
The ztimer_usec usage is optional and only used for debugging. I don't want to pull it in by default here.

Reply:
Ok